flocker_bb/builders/flocker_vagrant.py

```diff
@@ -340,11 +374,48 @@ def test_installed_package(box):
 from buildbot.locks import MasterLock
 
 
+# Dictionary mapping providers for client installation testing to a list
+# of sets of variants to test on each provider.
+@attributes([
+    Attribute('provider'),
+    # Vagrant doesn't take a distribution.
```

Comment:
In http://doc-dev.clusterhq.com/gettinginvolved/acceptance-testing.html#vagrant the distribution is explicit. Perhaps it should be the same here.